mlflow/tensorflow/__init__.py

```diff
@@ -129,36 +102,18 @@ def log_model(
     await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
     pip_requirements=None,
     extra_pip_requirements=None,
+    **kwargs,
 ):
     """
-    Log a *serialized* collection of TensorFlow graphs and variables as an MLflow model
-    for the current run. This method operates on TensorFlow variables and graphs that have been
-    serialized in TensorFlow's ``SavedModel`` format. For more information about ``SavedModel``
-    format, see the TensorFlow documentation:
-    https://www.tensorflow.org/guide/saved_model#save_and_restore_models.
-
-    This method saves a model with both ``python_function`` and ``tensorflow`` flavors.
-    If loaded back using the ``python_function`` flavor, the model can be used to predict on
-    pandas DataFrames, producing a pandas DataFrame whose output columns correspond to the
-    TensorFlow model's outputs. The python_function model will flatten outputs that are length-one,
-    one-dimensional tensors of a single scalar value (e.g.
-    ``{"predictions": [[1.0], [2.0], [3.0]]}``) into the scalar values (e.g.
-    ``{"predictions": [1, 2, 3]}``), so that the resulting output column is a column of scalars
-    rather than lists of length one. All other model output types are included as-is in the output
-    DataFrame.
-
-    :param tf_saved_model_dir: Path to the directory containing serialized TensorFlow variables and
-                               graphs in ``SavedModel`` format.
-    :param tf_meta_graph_tags: A list of tags identifying the model's metagraph within the
-                               serialized ``SavedModel`` object. For more information, see the
-                               ``tags`` parameter of the
-                               ``tf.saved_model.builder.SavedModelBuilder`` method.
-    :param tf_signature_def_key: A string identifying the input/output signature associated with the
-                                 model. This is a key within the serialized ``SavedModel`` signature
-                                 definition mapping. For more information, see the
-                                 ``signature_def_map`` parameter of the
-                                 ``tf.saved_model.builder.SavedModelBuilder`` method.
+    Log a Keras model.
+
+    :param model: The Keras model to be saved.
```

Comment:
@WeichenXu123 Can we also support TensorFlow Core APIs? https://www.tensorflow.org/guide/core/quickstart_core